composables/useCheckout.ts

```diff
@@ -38,10 +42,26 @@ export const useCheckout = () => {
   const load = async (checkoutId: String) => await getOrCreateCheckout({ checkoutId })
   const loadFromCart = async (cartId: String) => await getOrCreateCheckout({ cartId })
 
-  const setShippingInfo = async (variables) => {
+  const setPersonalInfo = async (variables) => {
     loading.value = true
     try {
       const response = await fetcher({
+        query: updateOrder,
+        variables,
+      })
+      checkout.value = response?.data?.checkout
```

Comment:
call `await load(checkout.value.id)` after the mutation instead

Reply:
now calling await load(checkout.value.id) after settingup personalInfo